Terminal.Gui/Views/TabView.cs

```diff
@@ -194,8 +194,8 @@ public override void Redraw (Rect bounds)
 				int spaceAtBottom = Math.Max (0, GetTabHeight (false) - 1);
 				int startAtY = Math.Max (0, GetTabHeight (true) - 1);
 
-				DrawFrame (new Rect (0, startAtY, bounds.Width,
-			       Math.Max (bounds.Height - spaceAtBottom - startAtY, 0)), 0, true);
+				DrawFrame (new Rect (0, startAtY, Bounds.Width,
```

Comment:
Should be able to revert this change once https://github.com/gui-cs/Terminal.Gui/pull/2309 is merged.